mozilla_version/firefox.py

```diff
@@ -43,7 +47,8 @@
 from mozilla_version.version import VersionType
 
 _VALID_VERSION_PATTERN = re.compile(r"""
-^(?P<major_number>\d+)\.(
+^(firefox-)?
```

Comment:
We probably want to look for `devedition` and `thunderbird` here as well.

That being said, I'm wary of parsing all the variant ways of expressing a version willy-nilly. I think we should support parsing both formats, but we should know at the point where we are parsing a version number which style it should be, and should just parse that style.

Reply:
Sounds good. I'll wait for the proposal you implemented yesterday :)

Reply:
Done. Latest commits now support `devedition`, `thunderbird`, and `fennec`